docs/aws/docker-compose.aws.yml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+volumes:
+  goerli-data:
+    driver_opts:
+      uid: 0
+      gid: 0
```

Comment:
Why are these required here?

What's the storage driver we're using here?

Reply:
AWS integration uses EFS storage. See the [ecs-integration doc](https://docs.docker.com/cloud/ecs-integration/#:~:text=To%20work%20around%20the%20possible%20conflict%2C%20you%20can%20set%20the%20volume%20uid%20and%20gid%20to%20be%20used%20when%20accessing%20a%20volume%3A) for this setup. I've tried first to put 1000 / 1000 as you do in the Dockerfile but it was not working and so sticked to the doc example.